apps/extension/client/src/initiateEditor.ts

```diff
@@ -91,6 +91,7 @@ async function getWebviewHtml(
     config,
     layoutString,
     implementations,
+    machineId: `vscode:${vscode.env.machineId}`,
```

Comment:
nit: It's a bit confusing that vs code calls this `machineId` since we already have this in our terminology. Should we call it `deviceId` on our end?

Reply:
Ah, I've missed that comment when taking a look at our internal PR in isolation. Yeah... this makes sense. Gonna adjust it in a second.